packages/react-ui-core/src/Ratings/RatingBar.js

```diff
@@ -58,6 +64,20 @@ export default class RatingBar extends PureComponent {
     )
   }
 
+  handleClick(event) {
```

Comment:
What does this need to be added for? Is this so you can click and select a rating? If not, I don't see a use case for this.